packages/fiori/test/samples/ViewSettingsDialog.sample.html

```diff
@@ -0,0 +1,46 @@
+<header class="component-heading">
+	<h2 class="control-header">ViewSettingsDialog</h2>
+	<div class="component-heading-since">
+		<span><!--since_tag_marker--></span>
+	</div>
+</header>
+<div class="component-package">@ui5/webcomponents-fiori</div>
+
+<div class="control-tag">&lt;ui5-view-settings-dialog&gt;</div>
+
+<section>
+	<h3>Usage</h3>
+	<div class="snippet">
+		<ui5-button id="btnOpenDialog1">Open ViewSettingsDialog</ui5-button>
+		<ui5-view-settings-dialog id="vsd1">
+			<ui5-li slot="sortItems">Name</ui5-li>
+			<ui5-li slot="sortItems">Position</ui5-li>
+			<ui5-li slot="sortItems">Company</ui5-li>
+			<ui5-li slot="sortItems">Department</ui5-li>
+		</ui5-view-settings-dialog>
+		<br />
+		<br />
+		<div id="vsdResults"></div>
+	</div>
+	<pre class="prettyprint lang-html"><xmp>
+		<ui5-view-settings-dialog>
+			<ui5-li slot="sortItems">Name</ui5-li>
+			<ui5-li slot="sortItems">Position</ui5-li>
+			<ui5-li slot="sortItems">Company</ui5-li>
+			<ui5-li slot="sortItems">Department</ui5-li>
+		</ui5-view-settings-dialog>
+	</xmp></pre>
+	<script>
+		var vsdResults = document.getElementById("vsdResults");
```

Comment:
It would be a good idea if you add this script inside the ```xmp``` tag in order to show it as plain text under the working sample